nvflare/client/flare_agent.py

```diff
@@ -343,16 +346,11 @@ def __init__(
 
 
 class FlareAgentWithFLModel(FlareAgent):
```

Comment:
Please move this class to a separate file so that the flare_agent.py will be more generic (and not importing those FLModel stuff).